crates/bitwarden-ipc/src/wasm/communication_backend.rs

```diff
@@ -68,12 +66,61 @@ impl JsCommunicationBackend {
     }
 }
 
-impl CommunicationBackend for JsCommunicationBackend {
-    type SendError = JsValue;
+impl From<&JsCommunicationBackend> for ThreadSafeJsCommunicationBackend {
```

Comment:
Minor thing, but it seems weird that a From impl would ever spawn a task, it feels like this would be better off as a separate function in  `JsCommunicationBackend`

Reply:
Yeah no that's fair, fixed!